src/rules.ts

```
@@ -0,0 +1,42 @@
+/// looks like some pattern here
+
+let rules = [
```

Biome

[error] This let declares a variable that is only assigned once.

'rules' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)